Doc/howto/descriptor.rst

```diff
@@ -432,10 +869,9 @@ Using the non-data descriptor protocol, a pure Python version of
         def __init__(self, f):
             self.f = f
 
-        def __get__(self, obj, klass=None):
-            if klass is None:
-                klass = type(obj)
+        def __get__(self, obj, cls=None):
+            if cls is None:
+                cls = type(obj)
             def newfunc(*args):
-                return self.f(klass, *args)
+                return self.f(cls, *args)
             return newfunc
```

Comment:
Instead of using `newfunc`, wouldn't it be better to return `types.MethodType(self.f, cls)` similar to what is done in `Function`?